core/modules/email/README.md

```diff
@@ -0,0 +1,23 @@
+Email Field
+======================
+
+This module provides an email field type. This project is a Backdrop CMS version of the [Email Field](https://www.drupal.org/project/email) Drupal module, created by Matthias Hutterer.
+
+Features
+---------
+
+  * Validation of emails
+  * Turns addresses into mailto links
+  * Contact form (see Display settings)
+  * Exposes fields to Views
+
+License
+--------
+
+This project is GPL v2 software. See the LICENSE.txt file in this directory for complete text.
+
+Maintainers
+------------
+
+Juan Olalla (https://github.com/juanolalla/)
+
```

Comment:
Let's remove this entire file. We don't include README.md files for core modules.